CauldronMods/Controller/Heroes/Gargoyle/CharacterCards/GargoyleCharacterCardController.cs

```diff
@@ -10,33 +10,170 @@ namespace Cauldron.Gargoyle
 {
     public class GargoyleCharacterCardController : HeroCharacterCardController
     {
+        private const int REDUCE_DAMAGE_POWER = 1;
+        private const int REDUCE_DAMAGE_INCAP = 2;
+        private const int INCREASE_DAMAGE_POWER = 1;
+        private const int INCREASE_DAMAGE_INCAP = 2;
+        private const int NUMBER_USES_ONCE = 1;
+
         public GargoyleCharacterCardController(Card card, TurnTakerController turnTakerController) : base(card, turnTakerController)
         {
         }
+
+        /*
+         * Power:
+         * "Select a target. Reduce the next damage it deals by 1. Increase the next damage Gargoyle deals by 1."
+         */
         public override IEnumerator UsePower(int index = 0)
         {
+            List<SelectTargetDecision> storedResults = new List<SelectTargetDecision>();
+            IEnumerable<Card> choices = base.FindCardsWhere(new LinqCardCriteria((Card c) => c.IsTarget && c.IsInPlayAndHasGameText));
+            ReduceDamageStatusEffect reduceDamageStatusEffect;
+            IncreaseDamageStatusEffect increaseDamageStatusEffect; 
+            IEnumerator coroutine;
+            Card selectedCard;
+
+            // Select a target. 
+            coroutine = base.GameController.SelectTargetAndStoreResults(base.HeroTurnTakerController, choices, storedResults, selectionType: SelectionType.None, cardSource: base.GetCardSource());
+            if (base.UseUnityCoroutines)
+            {
+                yield return base.GameController.StartCoroutine(coroutine);
+            }
+            else
+            {
+                base.GameController.ExhaustCoroutine(coroutine);
+            }
+
+            if (storedResults != null && storedResults.FirstOrDefault().SelectedCard != null)
+            {
+                selectedCard = storedResults.FirstOrDefault().SelectedCard;
+
+                // Reduce the next damage it deals by 1
+                reduceDamageStatusEffect = new ReduceDamageStatusEffect(REDUCE_DAMAGE_POWER)
```

Comment:
The nested constructor is from a bad decompile and shouldn't be used.  flatten out.
add effect.UntilTargetLeavesPlay(selectedCard);

Reply:
I'm not sure what you are referring to here. Are you talking about new ReduceDamageStatusEffect(*) {} or about SourceCriteria = { IsSpecificCard = selectedCard }?